examples/fine-tune-cli/src/main.rs

```diff
@@ -0,0 +1,143 @@
+use std::path::PathBuf;
+
+use async_openai::{
+    types::{CreateCompletionRequestArgs, CreateFileRequestArgs, CreateFineTuneRequestArgs},
+    Client,
+    config::OpenAIConfig,
+};
+use clap::{arg, Command};
+
+// TODO: Constructive error handling
+async fn data(paths: Vec<&PathBuf>, client: Client<OpenAIConfig>) {
+    if paths.len() > 2 {
+        println!("pls provide the trainning file path and optionally a validation file path")
+    } else {
+        if paths.len() < 2 {
+            let train_request = CreateFileRequestArgs::default()
+                .file(paths[0])
+                .purpose("fine-tune")
+                .build()
+                .unwrap();
+
+            let trainning_data = client.files().create(train_request).await.unwrap();
+
+            let fine_tune_request = CreateFineTuneRequestArgs::default()
+                .training_file(trainning_data.id)
+                .build()
+                .unwrap();
+
+            let job = client.fine_tunes().create(fine_tune_request).await.unwrap();
+
+            println!("Save the ft job ID: {:?}", job.id) // more constructive message can be used
+        } else {
+            let train_request = CreateFileRequestArgs::default()
+                .file(paths[0])
+                .purpose("fine-tune")
+                .build()
+                .unwrap();
+
+            let validate_request = CreateFileRequestArgs::default()
+                .file(paths[1])
+                .purpose("fine-tune")
+                .build()
+                .unwrap();
+
+            let trainning_data = client.files().create(train_request).await.unwrap();
+
+            let validation_data = client.files().create(validate_request).await.unwrap();
+
+            let fine_tune_request = CreateFineTuneRequestArgs::default()
+                .training_file(trainning_data.id)
+                .validation_file(validation_data.id)
+                .build()
+                .unwrap();
+
+            let job = client.fine_tunes().create(fine_tune_request).await.unwrap();
+
+            println!("Save the ft job ID: {:?}", job.id) // more constructive message can be used
+        }
+    }
+}
+
+async fn retrieve(job_id: String, client: Client<OpenAIConfig>) {
+    let ss = client.fine_tunes().retrieve(&job_id).await.unwrap();
+
+    if let Some(ft_model) = ss.fine_tuned_model {
+        println!("{:?}", ft_model)
+    } else {
+        println!("Please wait a while, your model is not done processing");
+    }
+}
+
+async fn completion(model: String, prompt: String, client: Client<OpenAIConfig>) {
+    let request = CreateCompletionRequestArgs::default()
+        .model(model)
+        .prompt(prompt)
+        .max_tokens(40_u16)
```

Comment:
```suggestion
        .max_tokens(1_u16)
```

For classification max_token =1 as per https://platform.openai.com/docs/guides/fine-tuning/classification

Otherwise it outputs many tokens like this:

```
Choice { text: ",positive groupnegative groups positive groupnegative group positive groupnegative grouppositive negativepositivepositivepositivepositivepositivenegativepositivenegativenegativenegativenegativepositivenegativenegativepositivenegativenegativepositivenegativepositivepositivenegativepositivepositive", index: 0, logprobs: None, finish_reason: Some("length") }

```